server/src/test/java/io/spine/server/bc/given/Given.java

```diff
@@ -53,4 +66,56 @@ public static BcProjectStarted projectStarted(ProjectId id) {
                                    .build();
         }
     }
+
+    public static class NoopCommandDispatcher implements CommandDispatcher<ProjectId> {
```

Comment:
We usually use `NoOp...` spelling, not `Noop`. I can see just a single `Noop` usage in the project (see `NoopQueryProcessor`), which also needs to be addressed (maybe you could help?)